src/fbrs/model/DatabaseManager.java

```diff
@@ -419,6 +421,24 @@ private int generateDarshKey(int userType) {
         return key;
     }
 
+    //This method will return 0 if the backup process completed successfully..
+    public int backup(String path) throws InterruptedException, IOException {
+        Runtime runtime = Runtime.getRuntime();
+        Process process = runtime.exec(" cmd /c pg_dump -d fbrsdb -U admin -h osama-radwan.com -f \"" + path + "\"", null,
+                new File("C:\\Program Files\\PostgreSQL\\12\\bin"));
```

Comment:
Replace with this
reader = ResourceBundle.getBundle("dbconfig");

fbrsdb -> reader.getString("db.databaseName")
admin -> reader.getString("db.username")
osama-radwan.com -> reader.getString("db.serverName")